server/lib/nl/detection.py

```diff
@@ -133,9 +133,8 @@ class ContainedInPlaceType(str, Enum):
   # Typically corresponds to county equivalent
   EU_NUTS_3 = "EurostatNUTS3"
 
-  # Across is a generic containedInPlaceType which determines if the
-  # query is using the word "across".
-  ACROSS = "Across"
+  # Indicates that the fulfiller should guess the child-type.
```

Comment:
nit: while this works, it feel a little "hacky", no? "Guess" is an action and not a containedin place type. Wouldn't it be better to map to some OTHER type which then triggers the guessing action?

Reply:
OTHER doesn't give any further info, but GUESS says something downstream should guess the type?  If we want it changed DEFAULT_CHILD_TYPE could be an option.

Let me know if you prefer that, and I can change in a follow on.